### PR TITLE
Welcome guide post-golive tweaks

### PR DESCRIPTION
**Pull request by peteryates**
### Trello card

https://trello.com/c/1CJXHlbS/2737-welcome-guide-tweaks-post-go-live

### Context

Some smallish content changes and tweaks.

### Changes proposed in this pull request

- Increase contrast on case study links
- Make 'we need teachers like you' text dynamic

### CODE REVIEW

File: app/views/content/welcome/landing/_exciting-times-ahead.html.erb [hunk 1]
```diff
@@ -31,6 +31,23 @@
...
       <div class="card">
         <div class="number">2</div>
+        <%= image_pack_tag("media/images/content/welcome-guide/person-paper.svg", alt: "") %>
+        <div class="stage-info">
+          <h3 class="stage-name">Check your qualifications.</h3>
+          <p>
+            If you’ve got a degree or equivalent qualification, you’re ready
+            for a teacher training courses.
+          </p>
+          <a href="/funding-your-training">
```
Comment (gemmadallmandfe) [inline app/views/content/welcome/landing/_exciting-times-ahead.html.erb:41]:
```suggestion
          <a href="/ways-to-train">
```

Comment (gemmadallmandfe) [reply app/views/content/welcome/landing/_exciting-times-ahead.html.erb:41]: I know Havas didn't supply a link but I think this is probably better going to Ways to train as that does talk about QTS, PGCE, etc.

**Comment by gemmadallmandfe** [comment]
Thanks @peteryates !

Two things:

- I'd suggest changing the link in the qualifications section?

- I can't get the generic subjects to appear in the `Teaching x is so reward` and `we need x teachers like you more than ever before` sections - I've tried a few different subjects in the Review app, including in icognitio

File: app/views/content/welcome/landing/_exciting-times-ahead.html.erb (hunk 1 above)
Comment (gemmadallmandfe) [inline app/views/content/welcome/landing/_exciting-times-ahead.html.erb:41]:
```suggestion
          <a href="/ways-to-train">
```

File: app/views/content/welcome/landing/_exciting-times-ahead.html.erb [hunk 2]
```diff
@@ -31,6 +31,23 @@
     <div class="stage">
       <div class="card">
         <div class="number">2</div>
+        <%= image_pack_tag("media/images/content/welcome-guide/person-paper.svg", alt: "") %>
+        <div class="stage-info">
+          <h3 class="stage-name">Check your qualifications.</h3>
+          <p>
+            If you’ve got a degree or equivalent qualification, you’re ready
+            for a teacher training courses.
```
Comment (gemmadallmandfe) [inline app/views/content/welcome/landing/_exciting-times-ahead.html.erb:39]:
```suggestion
            for a teacher training course.
```

**Comment by gemmadallmandfe** [comment]
Still an extra space on generic version before full-stop on 'My journey into teaching' (sorry - thought this was fixed already):

image

**Comment by gemmadallmandfe** [comment]
Looks good to me @peteryates - except the build has failed! Will approve as soon as this is sorted

**Comment by peteryates** [comment]
There was a timeout on one of the tests, reran and they worked 🤷🏽‍♂️

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me!

**Pull request merged**